pages/attendee/profile.js

```diff
@@ -84,60 +109,85 @@ function Profile() {
 
             <div className="grid-cols-2 overflow-hidden">
               <div className="col-span-1 float-left w-1/2 font-iregular">
-                💰170 Tokens
+                💰{user.token_balance} Tokens
               </div>
               <div className="col-span-1 float-left w-1/2 font-iregular">
-                🏅68 Badges
+                🏅{user.badge_count} Badges
```

Comment:
Maybe add a space between the emoji and the text, what do you think? 
```suggestion
                🏅 {user.badge_count} Badges
```